.github/workflows/multi-node.yml

```diff
@@ -73,8 +82,97 @@ jobs:
           body: |
             Multi node test of ${{github.repository}} failed!
             https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+      
       - name: Cleanup the environment
         if: ${{ always() }}
         shell: bash {0}
         run: |
           gcloud container clusters delete "akka-multi-node-${GITHUB_RUN_ID}" --quiet
+
+  akka-artery-aeron-cluster-tests:
+    name: Artery Aeron UDP Cluster
+    runs-on: ubuntu-20.04
+    if: github.repository == 'akka/akka'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Kubectl
+        run: |
+          sudo snap install kubectl --classic
+      - uses: google-github-actions/setup-gcloud@v0.2
+        with:
+          service_account_key: ${{ secrets.GKE_SA_KEY }}
+          project_id: ${{ secrets.GKE_PROJECT }}
+
+      - name: Create the cluster
+        run: |-
+          gcloud config set compute/region us-central1
+          gcloud config set compute/zone us-central1-c
+          ./kubernetes/create-cluster-gke.sh "akka-artery-aeron-cluster-${GITHUB_RUN_ID}"
+
+      - name: Setup Pods
+        run: |
+          # Stress tests are using 13 nodes
+          # on port 6000 for UDP
+          ./kubernetes/setup.sh 15 6000 multi-node-test.hosts
```

Comment:
all tests here use `akka.remote.artery.transport=aeron-udp` so we need to use port 6000 which is the UDP port that is configured in k8s